Godeps/Godeps.json

```diff
@@ -27,48 +27,48 @@
 		},
 		{
 			"ImportPath": "github.com/GoogleCloudPlatform/kubernetes/pkg/api",
-			"Comment": "v0.7.0-256-g7dec65f",
-			"Rev": "7dec65f535ad95cdc1f88a672d6880f0720c8241"
+			"Comment": "v0.7.0-13-gff7eb21",
```

Comment:
How do we tell if v0.7.0-13-* is moving forward from v0.7.0-256? Shouldn't it be v0.7.2-smth?